docs/source/methods/geometry.rst

```diff
@@ -892,6 +896,71 @@ Dxy + Eyz + Fxz + Gx + Hy + Jz + K = 0`. Thus, the gradient to the surface is
     \\ 2Cz + Ey + Fx + J \end{array} \right ).
 
 
+.. _white:
+
+------------------------------
+White Boundary Conditions
+------------------------------
+
+The white boundary condition is usually applied in deterministic codes, where the particle
+will hit the surface and travel back with isotropic angular distribution. The change in 
+particle's direction is sampled from a cosine distribution instead of uniform. 
+Figure :num:`fig-cosine-dist` shows an example of cosine-distribution reflection on the 
+arbitrary surface relative to the surface normal.
+
+.. _fig-cosine-dist:
+
+.. figure:: ../_images/cosine-dist.png
```

Comment:
Did you create this figure yourself? If not, I just want to make sure that we give proper attribution to wherever it came from.

Reply:
I creted this figure by myself. However, the idea is coming from those two papers
[1]. She, Ding. "On the white boundary condition in Monte Carlo simulation." Annals of Nuclear Energy 132 (2019): 413-414. https://doi.org/10.1016/j.anucene.2019.05.006
[2]. Mallick, Amod Kishore, Indrajeet Singh, and P. D. Krishnani. "Incorporation of White and Periodic Boundary Conditions for Geometry Modeling in Monte Carlo Code PATMOC." (2015). DOI: 10.13140/RG.2.1.4378.6405

Reply:
Great! I really like the figure. Should one of the vectors (in red) be v' though? The figure shows both as v right now.

Reply:
Um, Actually I am goint to show that the scale of velocity does not changed. Although the vector (scale + direction) changes its direction. If I use v', should I add the vector arrow on the top just like \omega?

Reply:
I see; in that case I think it's fine as is. Thanks!

Reply:
Great. Thanks too.